reference/clever-tools/getting_started.md

```diff
@@ -48,39 +48,39 @@ apt install apt-transport-https ca-certificates
 ```
 
 ```sh
-apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys "379CE192D401AB61"
-echo "deb https://dl.bintray.com/clevercloud/deb stable main" | tee -a /etc/apt/sources.list.d/clevercloud.list
+apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys "A7B8F38C536F1DF2"
+echo "deb https://nexus.clever-cloud.com/repository/deb bionic stable main" | tee -a /etc/apt/sources.list
 apt update
 apt install clever-tools
 ```
 
 NOTES:
 
-* The `.deb` packages are hosted on Bintray (their GPG key is required to trust their signed packages).
+* The `.deb` packages are hosted on Clever Cloud's public Nexus instance available at [https://nexus.clever-cloud.com](https://nexus.clever-cloud.com) (their GPG key is required to trust their signed packages).
```

Comment:
```suggestion
* The `.deb` packages are hosted on Clever Cloud's public Nexus instance available at [https://nexus.clever-cloud.com](https://nexus.clever-cloud.com) (their PGP key is required to trust their signed packages).
```

`GPG` is the GNU implementation of `PGP`. Even though I'm not sure what this sentence means, who is "their"?